hack/init_postgres.sql

```diff
@@ -87,7 +105,8 @@ VALUES  ('20171026211738'),
         ('20180103212743'),
         ('20180108183307'),
         ('20180119214651'),
-        ('20180125194653');
+        ('20180125194653'),
+		('20211120104808');
```

Comment:
Likewise, we don't need to add this too because the go framework we're using automatically updates the `schema_migrations` table with all the migrations applied. When a new migration is created and gotrue is restarted, the framework refers to the `schema_migrations` table to see which migrations have been applied already and only applies those that come after the latest migration in the table.